src/spotify.c

```diff
@@ -2042,7 +2044,7 @@ void
 spotify_oauth_callback(struct evbuffer *evbuf, struct evkeyvalq *param, const char *redirect_uri)
 {
   const char *code;
-  const char *err;
+  const char *err = "";
```

Comment:
Is this required? If there are code paths where spotifywebapi_token_get() returns < 0 without setting err it should be fixed. For safety, spotifywebapi_token_get() should probably also set err to "" to start with.